api/app/lib/updaters.rb

```diff
@@ -102,7 +102,7 @@ def update_belongs_to(model, name, to_add)
   def update_has_many(model, name, to_add)
     value = relationship_map(model, name, to_add)
     model.send("#{name}=", value)
-    model.send(name.to_s).try(:sort, value)
+    model.send(name.to_s).try(:sort_makers, value)
```

Comment:
I don't think this is a prudent change. Sure, it will make it work for this one case, but it's going to break sorting on has_many relationships for every other type of association. This update class provides some automation and abstraction around updating models from params — as such, only relatively generic method calls can exist here.

If you reall need maker-specific code, you should override the update_has_many method in  Updaters::Maker.